pkg/sql/logictest/testdata/logic_test/alter_column_type

```diff
@@ -556,3 +556,13 @@ INSERT INTO regression_54844 VALUES (-9223372036854775807)
 
 statement error integer out of range for type int2
 ALTER TABLE regression_54844 ALTER COLUMN i TYPE int2
+
+# Regression test for issue #71089
```

Comment:
drive by comment: if you're optimising for backports, you should probably move this up somewhere higher